Move inputsList to  PaginatedEntityDataTable

ousmaneo opened the pull request:
This PR moves the InputsList to PaginatedDataTable

Part of https://github.com/Graylog2/graylog-plugin-enterprise/issues/12248

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have requested a documentation update.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

laura-b-g commented on graylog2-web-interface/src/components/inputs/InputsOveriew/cells/TitleCell.tsx:
```diff
@@ -0,0 +1,38 @@
+};
+
+const TitleCell = ({ input }: Props) => (
+  <span>
```
Nitpick: Do we need this span?

ousmaneo commented:
Thank you for your review.
> Code-wise it looks good, just some nitpicks.
> 
> I found some issues while testing:
> 
> * when a new input is launched, the list is not updated, only after reload

Fixed it.

> * State filter shows a date picker
> 
> <img alt="Screenshot 2025-11-13 at 12 21 52" width="1270" height="759" src="https://private-user-images.githubusercontent.com/197286649/513860907-c1e7f533-3f15-44dc-978a-9a714feb36ab.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMwNDAyMzEsIm5iZiI6MTc2MzAzOTkzMSwicGF0aCI6Ii8xOTcyODY2NDkvNTEzODYwOTA3LWMxZTdmNTMzLTNmMTUtNDRkYy05NzhhLTlhNzE0ZmViMzZhYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExM1QxMzE4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMzA2NjVmNjJhODVkZjNhYmE0YTQzZmI0ZWQ4YTlmZWZiMGU4NTk5MTlkNjhhZGM1MGE4M2FiZTQyNTdlZjRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Q4jvJjWZPm9djbRxn3q4WhYflsGcb-J9lepXzTBqPz4">

This is because the backend is returning the wrong format, and it will be fixed in a subsequent PR.

> * Global and node filter could have predefined values
> * Input State badges not shown

These will also be fixed in an additional PR once the backend is fixed. The backend filters are not in the right shape.

> * I would add an icon to the title column that indicates that it is collapsible. I only accidently found out

I discussed this with Tully, and we decided to make the whole row clickable to show the expanded section.

> * For the extended column: Can we move static fields below the Throughput/Metrics section?

The expanded section will be reorganized a bit in another PR once the backend for extractors is ready.

> * The static fields table looks wrong, also, deleting an item does not work

Yes this is also because the backend is returning the wrong formats. Maxwell is working on it

> <img alt="Screenshot 2025-11-13 at 12 52 20" width="440" height="122" src="https://private-user-images.githubusercontent.com/197286649/513860617-bfb9a073-5b69-42d1-9baa-7d48f839e76c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMwNDAyMzEsIm5iZiI6MTc2MzAzOTkzMSwicGF0aCI6Ii8xOTcyODY2NDkvNTEzODYwNjE3LWJmYjlhMDczLTViNjktNDJkMS05YmFhLTdkNDhmODM5ZTc2Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExM1QxMzE4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YTRiOTU0NTE4OGUxYjRiMzhiMzllODRmNzU1OWQyN2ExMDY3MzNiMjc0MzMwYjAwYTQ2YzIwZDI4NzBmOGQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.YFkeh6UpqCZJOXirlZS302OkzXQAhUamI8oJI2LhVZU"> <img alt="Screenshot 2025-11-13 at 12 52 50" width="2000" height="803" src="https://private-user-images.githubusercontent.com/197286649/513860655-bc6e3650-3247-4e07-b7a3-3db10fc2f132.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMwNDAyMzEsIm5iZiI6MTc2MzAzOTkzMSwicGF0aCI6Ii8xOTcyODY2NDkvNTEzODYwNjU1LWJjNmUzNjUwLTMyNDctNGUwNy1iN2EzLTNkYjEwZmMyZjEzMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExM1QxMzE4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMzg2YTY0NTNkMjVlYTZjOTkyMzNiODllMGY5YWQzZDRhYTBhZWQwMTMwNDE4ODdhYjM4YmIxYTZiOGVjZmUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.RriWElVIN3JbVRW530ex8Eja2LNTH2-fxq09FccNuP0">
> * Node and Global cells are empty

These need the backend to be fixed first so that it will be done in a follow up PR

> * In the columns selection dropdown I can select a second, empty State cell

Fixed

> * When I run more than one node all state badges are yellow
> 
> <img alt="Screenshot 2025-11-13 at 13 14 45" width="2000" height="264" src="https://private-user-images.githubusercontent.com/197286649/513860473-ca22700e-d31b-48ef-883f-0c08e55c574d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMwNDAyMzEsIm5iZiI6MTc2MzAzOTkzMSwicGF0aCI6Ii8xOTcyODY2NDkvNTEzODYwNDczLWNhMjI3MDBlLWQzMWItNDhlZi04ODNmLTBjMDhlNTVjNTc0ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExM1QxMzE4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMGMxNjUwNTRjMjYxYzQyYTU1OTI1NzFlNzAwYjczMzdhNGZmZWQ2N2MxOGM1Y2I1M2QxOWY5NWVkNDk2MGYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7d24bomCwPMpBWFCh3kWEa_MKcOns0uZP82E8Jdjnx0">

The badge will default to warning if the input is failing on one of the nodes

> * There are no metrics shown for e.g. a running Random HTTP input
> 
> <img alt="Screenshot 2025-11-13 at 13 16 48" width="2000" height="265" src="https://private-user-images.githubusercontent.com/197286649/513860419-6e2480b5-a57a-4e4e-a945-5e6dc1b3e900.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMwNDAyMzEsIm5iZiI6MTc2MzAzOTkzMSwicGF0aCI6Ii8xOTcyODY2NDkvNTEzODYwNDE5LTZlMjQ4MGI1LWE1N2EtNGU0ZS1hOTQ1LTVlNmRjMWIzZTkwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExM1QxMzE4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lM2YzODBjNzZhOGIzYzBhMWMzYzFmMTZkMmVkMjllODdkM2I3NTUwZjk5NjA5MmU0MmY0Y2ZiY2MxYzk3NDU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JJ3_3goXQvGqg3cjs-WrwIvYh0D6DfMMI0P-xxAF5sE">

That's normal because The Random HTTP input only has **msg/s Last Minute**
